lib/api/completeMultipartUpload.js

```diff
@@ -112,6 +115,12 @@ function completeMultipartUpload(authInfo, request, log, callback) {
         return callback(queryContainsVersionId);
     }
 
+    // Reconstruct mpuOverviewKey to serve
+    // as key to pull metadata originally stored when mpu initiated
+    const mpuOverviewKey =
```

Comment:
yeah, I fixed this in my other MPU branch but forgot to fix it here >.< thanks for catching